domain/src/main/java/org/oppia/domain/exploration/ExplorationRetriever.kt

```diff
@@ -32,39 +20,23 @@ class ExplorationRetriever @Inject constructor(
   // TODO(#169): Force callers of this method on a background thread.
   /** Loads and returns an exploration for the specified exploration ID, or fails. */
   internal fun loadExploration(explorationId: String): Exploration {
-    return when (explorationId) {
-      TEST_EXPLORATION_ID_0 -> loadExplorationFromAsset("test_exp_id_0.json")
-      TEST_EXPLORATION_ID_1 -> loadExplorationFromAsset("test_exp_id_1.json")
-      TEST_EXPLORATION_ID_2 -> loadExplorationFromAsset("test_exp_id_2.json")
-      TEST_EXPLORATION_ID_3 -> loadExplorationFromAsset("test_exp_id_3.json")
-      TEST_EXPLORATION_ID_4 -> loadExplorationFromAsset("test_exp_id_4.json")
-      FRACTIONS_EXPLORATION_ID_0 -> loadExplorationFromAsset("fractions_exploration0.json")
-      FRACTIONS_EXPLORATION_ID_1 -> loadExplorationFromAsset("fractions_exploration1.json")
-      RATIOS_EXPLORATION_ID_0 -> loadExplorationFromAsset("ratios_exploration0.json")
-      RATIOS_EXPLORATION_ID_1 -> loadExplorationFromAsset("ratios_exploration1.json")
-      RATIOS_EXPLORATION_ID_2 -> loadExplorationFromAsset("ratios_exploration2.json")
-      RATIOS_EXPLORATION_ID_3 -> loadExplorationFromAsset("ratios_exploration3.json")
-      else -> throw IllegalStateException("Invalid exploration ID: $explorationId")
-    }
+    val explorationObject =
+      jsonAssetRetriever.loadJsonFromAsset("$explorationId.json")
+        ?: return Exploration.getDefaultInstance()
+    return loadExplorationFromAsset(explorationObject)
   }
 
   // Returns an exploration given an assetName
-  private fun loadExplorationFromAsset(assetName: String): Exploration {
-    try {
-      val explorationObject =
-        jsonAssetRetriever.loadJsonFromAsset(assetName) ?: return Exploration.getDefaultInstance()
-      return Exploration.newBuilder()
-        .setId(explorationObject.getString("exploration_id"))
-        .setTitle(explorationObject.getString("title"))
-        .setLanguageCode(explorationObject.getString("language_code"))
-        .setInitStateName(explorationObject.getString("init_state_name"))
-        .setObjective(explorationObject.getString("objective"))
-        .putAllStates(createStatesFromJsonObject(explorationObject.getJSONObject("states")))
-        .build()
-    } catch (e: IOException) {
-      exceptionLogger.logException(e)
```

Comment:
Could we remove `exceptionLogger` from the constructor `private val exceptionLogger: ExceptionLogger`,  as we are no more using it?

Reply:
Removed.

Reply:
Thanks